awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -110,7 +109,7 @@ class JobTemplateForm extends Component {
           ...newLabels,
           {
             name: label,
-            organization: template.summary_fields.inventory.organization_id,
+            organization: loadedLabels[0].organization,
```

Comment:
`loadedLabels` may be empty, in which case this will result in a traceback.

Reply:
Taking a step back, I see that we're having to check whether or not `label` is a string before knowing how to handle it. Ideally, this handler should only need to worry about working with one data type. Under what conditions is `handleNewLabel` passed a string instead of an object?

Reply:
@jakemcdermott when sending newly associated labels to the api if it one that has been generated by the user then it needs `{name: 'New Label, organization_id: 1}` whereas if it's a label they clicked on that was already generated which is `{associated: true, id: 2]` if we store the labels in an array as strings, how do I distinguish between generated labels and associated labels?

Reply:
ah nvmd I got it.  👍